DecipherNow/gm-fabric-dashboard
# 1300 cleanup readme

Resolves #1300

## How to Test

  - Proofread the README
  - Follow the Quick Start instructions and make sure it works as expected (I'll check the Windows instructions as home this evening)
  - Make sure links resolve
